### DISCUSSION TITLE
How to get the galaxy brain achievement in 2024?

### DISCUSSION THREAD

**Discussion by ak-sohag** [community/community#143321] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How to get the galaxy brain achievement in 2024?

**Comment by ak-sohag** [comment]
To earn the Galaxy Brain achievement on GitHub, you need to provide valuable answers in GitHub Discussions. Specifically, this badge requires at least two of your answers to be marked as “accepted” by the original poster or a user with appropriate permissions. When you reach two accepted answers, you’ll receive the base Galaxy Brain badge, with higher badge tiers available for more accepted answers. Keep in mind that discussions must be on public repositories, and self-marked answers don’t count.

However, as of February 2024, achievements are no longer available in GitHub Community Discussions due to high volumes of low-quality posts. You can still earn this badge in other repository discussions outside of the main GitHub Community forum.

**Comment by harshakalluri1403** [comment]
To earn the "Galaxy Brain" badge on GitHub in 2024, you need to approve a pull request that has been open for an extended period. Look for pull requests in projects you contribute to that may have been overlooked or need revisiting. Approving these longstanding PRs showcases a keen understanding of the project and recognition of valuable contributions that may have been missed, demonstrating your thoroughness and insight as a collaborator.

**Comment by ebndev** [comment]
Hi @ak-sohag, thanks for being a part of the GitHub Community!

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. As you are aware, we made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. Self-marking answers falls under this umbrella as well.

As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. 

Thanks for understanding and we hope to see you participate in other discussions in the future!
